go/cmd/yolo/main.go

```diff
@@ -25,19 +27,27 @@ import (
 	_ "github.com/jinzhu/gorm/dialects/sqlite"
 	circleci "github.com/jszwedko/go-circleci"
 	"github.com/peterbourgon/diskv"
-	ff "github.com/peterbourgon/ff/v2"
-	"github.com/peterbourgon/ff/v2/ffcli"
+	ff "github.com/peterbourgon/ff/v3"
 )
 
-var (
-	verbose        bool
-	logFormat      string
-	dbStorePath    string
-	withPreloading bool
-)
+type GlobalOptions struct {
+	verbose     bool
+	logFormat   string
+	dbStorePath string
+
+	server server
+}
+
+func (g *GlobalOptions) commonFlagsBuilder(fs *flag.FlagSet) {
+	fs.BoolVar(&g.verbose, "v", false, "increase log verbosity")
+	fs.StringVar(&g.logFormat, "log-format", "console", strings.Join(zapconfig.AvailablePresets, ", "))
+	fs.StringVar(&g.dbStorePath, "db-path", ":memory:", "DB Store path")
```

Comment:
```suggestion
	fs.BoolVar(&g.verbose, "v", g.verbose, "increase log verbosity")
	fs.StringVar(&g.logFormat, "log-format", g.logFormat, strings.Join(zapconfig.AvailablePresets, ", "))
	fs.StringVar(&g.dbStorePath, "db-path", g.dbStorePath, "DB Store path")
```

And then, you add a `g.applyDefaults()` that you call at the beginning of the `func main()`.